src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -332,9 +332,17 @@
      *                              that don't use retries. For calls that use retries,
      *                              set the timeout in RetryOptions.
      *                              Default: 30000 (30 seconds)
-     *     @@type string $appName The codename of the calling service. Default 'gax'.
-     *     @@type string $appVersion The version of the calling service.
-     *                              Default: the current version of GAX.
+     *     @@type string $libName The API library software used for calling the service.
+     *                            (Unless you are writing an API client itself, leave this as default.)
+     *     @@type string $libVersion The API library software version used for calling the service.
+     *                               (Unless you are writing an API client itself,
+     *                               leave this as default.)
+     *     @@type string $appName Currently unsupported - reserved for future use.
+     *                            The name of the application calling the service. Recommended
+     *                            for analytics purposes.
+     *     @@type string $appVersion Currently unsupported - reserved for future use.
```

Comment:
This was mirroring Python's choice to keep them in the surface: https://github.com/googleapis/api-client-staging/blob/master/generated/python/gapic-google-cloud-pubsub-v1/google/cloud/gapic/pubsub/v1/publisher_client.py#L140

This may be unnecessary, as we will not break anyone if they keep using appName and appVersion (but the info will be dropped from the header) while python would have an unknown keyword arg if they removed.